packages/main/src/RPA/Images.py

```diff
@@ -163,35 +163,38 @@ def __init__(self):
         self.logger = logging.getLogger(__name__)
         self.matcher = TemplateMatcher(opencv=HAS_OPENCV)
 
-    def take_screenshot(self, filename=None, region=None, save_format="PNG"):
+    def take_screenshot(self, filename=None, region=None) -> Image:
         """Take a screenshot of the current desktop.
 
         :param filename:    Save screenshot to filename
         :param region:      Region to crop screenshot to
-        :param save_format: File format to save the screenshot in
         """
+
         region = to_region(region)
 
-        if region is not None:
-            image = ImageGrab.grab(bbox=region.as_tuple())
-        else:
-            image = ImageGrab.grab()
+        with mss.mss() as sct:
+            if region is not None:
+                image = sct.grab(region.as_tuple())
```

Comment:
Is the origin the top-left of the main monitor or the whole virtual screen?

Reply:
I am pretty sure the origin is the top-left of the whole virtual screen. mss docs were a bit lean on this so I'm unsure, will try to test